hivemind/averaging/averager.py

```diff
@@ -109,7 +109,7 @@ class DecentralizedAverager(mp.Process, ServicerBase):
     """
 
     _matchmaking: Matchmaking
-    _pending_group_assembled: asyncio.Event
+    _pending_groups_registered: asyncio.Event
```

Comment:
Note: changes to this file are required so that we can run multiple AllreduceRunners per step (for powerSGD and more in future)